ffbs-parker-nodeconfig/files/usr/sbin/nodeconfig.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/busybox sh
+# !!!!!!
+# THIS SCRIPT MUST HAVE GID 800 (gluon-mesh-vpn)
+# USE `gluon-wan nodeconfig.sh`
+# !!!!!!
+SIGN_PUB_KEY=/etc/parker/node-config-pub.key
+DEFAULT_SLEEP=20
+export LIBPACKETMARK_MARK=1
+
+tmpdir=/tmp/ff-Ohb0ba0u/
+mkdir -p "$tmpdir"
+
+version="$(cat /lib/gluon/release)"
+
+LOGGER="logger -s -t nodeconfig.sh"
+$LOGGER "Starting up. I am running ${version}"
+
+if [ ! -s /etc/parker/wg-privkey ]; then
+    $LOGGER No Wireguard keys found, generating...
+    mkdir -p /etc/parker
+    rm -f /etc/parker/wg-pubkey
+    wg genkey > /etc/parker/wg-privkey.tmp
+    wg pubkey < /etc/parker/wg-privkey.tmp > /etc/parker/wg-pubkey
+    sync
+    mv /etc/parker/wg-privkey.tmp /etc/parker/wg-privkey
+    $LOGGER Wireguard keys generated
+else
+    $LOGGER Wireguard keys found, proceeding
+fi
+
+pubkey=$(sed 's/+/%2B/g' /etc/parker/wg-pubkey)
+
```

Comment:
follow-up of the above suggested change to move the pubkey to uci
```suggestion

if ! uci -q get parker.nodeconfig.config_pubkey > "${SIGN_PUB_KEY}"; then
    $LOGGER unable to read or write pubkey from uci
fi
```

Reply:
@SmithChart still open

Reply:
See https://github.com/SmithChart/community-packages/pull/5

Reply:
@grische can you verify that this has been resolved by https://github.com/SmithChart/community-packages/pull/5 ?